LondonBritishLibrary/orient/BLorient758.xml

```diff
@@ -31,22 +31,21 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         </altIdentifier>
                         <altIdentifier>
                             <idno>Wright 332</idno>
-                            
+
                         </altIdentifier>
-                        
+
                     </msIdentifier>
                     <msContents>
                         <summary/>
-                        
-                        
-                        
+                        <msItem xml:id="ms_i1">
+                          <title type="complete" ref="LIT1948Mashaf"></title>
+                          <textLang mainLang="gez"></textLang>
+                          <colophon xml:id="coloph1" xml:lang="gez"><locus from="143a"></locus>
+                          <seg part="I">ዛቲ፡ መጽሐፍ፡ <add>ዘአባ፡ ወልደ፡ ስምዖን፡</add> ዘተዓገላ፡ ወዘሠረቃ፡</seg>
```

Comment:
This does not seem to be what we understand by colophon...of course, now it is difficult to say if it is an addition or not...I would maybe tentatively go with explicit here.
And could you create a person ID for Walda Səmʿon?

Reply:
I think it is not an addition, it was classified as a colophon by Wright, and it is just the very beginning, no idea what follows...

Reply:
I think that is terminology is just different than ours. In my experience, it would not be typical for a condemnation to be followed by something on the production of the manuscript (the other way round would be more usual), but of course we cannot know.

Reply:
I see, I went for excplicit for clarity

Reply:
It is always difficult :)